src/Symfony/Component/Uid/Ulid.php

```diff
@@ -114,6 +114,13 @@ public function toBinary(): string
         return hex2bin($ulid);
     }
 
+    /**
+     * Returns the identifier as a base32 case insensitive string.
+     *
+     * @see https://tools.ietf.org/html/rfc4648#section-6
+     *
+     * @example 09EJ0S614A9FXVG9C5537Q9ZE1 (len=26)
```

Comment:
this is not needed as it should inherit from the parent class.

Reply:
I adeded them because it looks like vscode does not do that by default :( 

![image](https://github.com/symfony/symfony/assets/408368/277859e2-226e-444a-8907-9cb3417f9ddb)